pkg/tsdb/cloudwatch/models/settings_test.go

```diff
@@ -71,4 +72,39 @@ func Test_Settings_LoadCloudWatchSettings(t *testing.T) {
 		assert.Equal(t, "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY", s.SecretKey)
 		assert.Equal(t, "AKIAIOSFODNN7EXAMPLE", s.AccessKey)
 	})
+	t.Run("Should set logsTimeout to default duration if it is not defined", func(t *testing.T) {
```

Comment:
Not related to your changes, but the original tests could use a test covering the unmarshal json error case

```golang
t.Run("Should return error for invalid json", func(t *testing.T) {
		settings := backend.DataSourceInstanceSettings{
			ID: 33,
			JSONData: []byte(`{
			"authType": "arn",
			"assumeRoleArn": "arn:aws:iam::123456789012:role/grafana",
			"logsTimeout": fluffles
		  }`),
			DecryptedSecureJSONData: map[string]string{
				"accessKey": "AKIAIOSFODNN7EXAMPLE",
				"secretKey": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
			},
		}

		_, err := LoadCloudWatchSettings(settings)

		assert.Error(t, err)
	})
```